lib/pxutil.js

```diff
@@ -267,9 +267,21 @@ function getTokenObject(cookie, delimiter = ':') {
     return { key: '_px3', value: cookie };
 }
 
+function isGraphql(req, config) {
+    if (req.method.toLowerCase() !== 'post') {
+        return false;
+    }
+
+    const routes = config['GRAPHQL_ROUTES'];
+    if (!Array.isArray(routes)) {
+        config.logger.error('Invalid configuration px_graphql_routes');
+        return false;
+    }
+    return routes.some(r => new RegExp(r).test(req.originalUrl));
```

Comment:
Now this won't work as originalUrl can contain query params etc, lets compare with the exact path